kustomize/overlays/dev/deployment.yaml

```diff
@@ -72,6 +72,8 @@ spec:
               value: "3"
             - name: POD_OWNER_KIND
               value: "deployment"
+            - name: DEBUG
+              value: "false"
```

Comment:
but this is for the dev env, should it be set to true? i guess we can toggle it, but devs won't know this thing exists

Reply:
I just left it as a place holder for dev when I was testing this.  Made it 2% easier to toggle :)